database/migrations/2022_04_19_054357_create_kata_string_end_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateKataStringEndTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('kata7_string_end', function (Blueprint $table) {
+            $table->id();
```

Comment:
Just an FYI, but i think $table supports fluent interface, so you could also do this:

`$table->id()
->string('text', 64)
->string('text_ending', 30)
->timestamps();`

Reply:
Noted!